one-pager/orchester/main.py

```
@@ -0,0 +1,102 @@
+        # Eliminar archivo local después de subirlo
+        if os.path.exists(pdf_title):
+            os.remove(pdf_title)
+        s = pyshorteners.Shortener()
+
+        return {"url":s.tinyurl.short(gcs_url)}, 200
+
+    except requests.exceptions.HTTPError as e:
+        error_details = e.response.text if e.response else "Sin detalles de respuesta."
+        print(f"Error HTTP al llamar a la función downstream: {e}\nDetalles: {error_details}")
+        return {
+            "error": f"Error al llamar a la función de destino: {str(e)}",
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)